wiki/Tournaments/UKCC/3/en.md

```diff
@@ -17,4 +44,658 @@ The **UK Community Cup 3** (***UKCC3***) was a British 1v1 double-elimination os
+| Mbarris ![][flag_GB] | 1 | **3** | ![][flag_GB] **Summer Solstice** |  |
+| **Cornfield** ![][flag_GB] | **0** | -1 | ![][flag_GB] Revios | *forfeit* |
+| **TopSp1n** ![][flag_GB] | **3** | 1 | ![][flag_GB] Feliciaforever |  |
+| **icecream-chan** ![][flag_GB] | **0** | -1 | ![][flag_GB] Kixs Student | *forfeit* |
+| Smiffi ![][flag_GB] | -1 | **0** | ![][flag_GB] **\[ B E A S T \]** | *forfeit* |
+| Ali161102 ![][flag_GB] | 1 | **3** | ![][flag_GB] **Sam** |  |
+| Hii ![][flag_GB] | 1 | **3** | ![][flag_GB] **Snookie** |  |
+| **Linkle** ![][flag_GB] | **3** | 0 | ![][flag_GB] - Sakura |  |
+| **Kardet** ![][flag_GB] | **0** | -1 | ![][flag_GB] RojasowyPL | *forfeit* |
+| Toadstool ![][flag_GB] | 0 | **3** | ![][flag_GB] **Claus** |  |
+| **Bae-** ![][flag_GB] | **3** | 0 | ![][flag_GB] Ik-rai |  |
+| Xim ![][flag_GB] | -1 | **0** | ![][flag_GB] **Darkdog** | *forfeit* |
+| Taiitoo ![][flag_GB] | -1 | **0** | ![][flag_GB] **MagicMantwon** | *forfeit* |
+| Hi Im Adam ![][flag_GB] | 0 | **3** | ![][flag_GB] **Jerome** |  |
+| L-A-M-E-Y \[ B \] ![][flag_GB] | 0 | **3** | ![][flag_GB] **L britt** |  |
+| Giblit ![][flag_GB] | 1 | **3** | ![][flag_GB] **xFluffyo** |  |
+| MySliderBreak ![][flag_IE] | 0 | **3** | ![][flag_GB] **jesse1412** |  |
+| Nega ![][flag_GB] | **3** | 0 | ![][flag_GB] xPray |  |
+| **AlexDark69** ![][flag_GB] | **3** | 0 | ![][flag_GB] LosslessAudio |  |
+| **derrik** ![][flag_GB] | **0** | -1 | ![][flag_GB] meltz | *forfeit* |
+| Tironas ![][flag_LT] | 1 | **3** | ![][flag_GB] **Doughy** |  |
+| Electovoid ![][flag_GB] | -1 | **0** | ![][flag_GB] **- Akemi** | *forfeit* |
+| **Orenjijusu** ![][flag_GB] | **3** | 0 | ![][flag_GB] Mashihoe |  |
+| **D u c k y** ![][flag_IE] | **3** | 0 | ![][flag_GB] \_Oliver\_ |  |
+
+Sunday, 2 July 2017:
+
+| Player 1 |  |  | Player 2 | Match info |
+| --: | :-: | :-: | :-- | :-- |
+| **Volcanism** ![][flag_GB] | **3** | 0 | ![][flag_GB] hypercyte |  |
+| **RndGuy** ![][flag_GB] | **0** | -1 | ![][flag_GB] Darthray | *forfeit* |
+| **Proteus** ![][flag_GB] | **0** | -1 | ![][flag_GB] ShadowTheMLGPro | *forfeit* |
+| **001thefish** ![][flag_GB] | **0** | -1 | ![][flag_DE] senpaiEra | *forfeit* |
+| **sonofgeek** ![][flag_GB] | **0** | -1 | ![][flag_GB] Jimzzyyy | *forfeit* |
+| **october** ![][flag_GB] | **3** | 0 | ![][flag_GB] Waluigi |  |
+| **Cephus** ![][flag_GB] | **0** | -1 | ![][flag_GB] koiishi | *forfeit* |
+| skaytan ![][flag_GB] | 1 | **3** | ![][flag_GB] **Mykaterasu** |  |
+| **SteffoST** ![][flag_GB] | **0** | -1 | ![][flag_GB] Kayneinn | *forfeit* |
+| SHUNIX ![][flag_GB] | -1 | **0** | ![][flag_GB] **PsychoStatic** | *forfeit* |
+| **Savag3blow** ![][flag_GB] | **3** | 1 | ![][flag_GB] -Blue- |  |
+| **Zainny** ![][flag_GB] | **0** | -1 | ![][flag_GB] Chug | *forfeit* |
+| **Chugger** ![][flag_GB] | **3** | 0 | ![][flag_GB] Respirte |  |
+| **djsettle** ![][flag_GB] | **3** | 0 | ![][flag_GB] Stedoss | *djsettle advances to group stage* |
```

Comment:
Not sure what the best way to handle this is, it doesn't feel right to include a blank match in the RO64 because the match never happened but it also feels a bit awkward to include it in the match info here since it doesn't directly involve this match up.

Context: djsettle was allowed a bye through the RO64 into the group stage as both potential opponents no-showed their RO128 match.

Reply:
wouldn't sth like this be fine?

```suggestion
| **djsettle** ![][flag_GB] | **0** | -1 | ![][flag_GB] Stedoss | *win by default* |
```

Reply:
No, because djsettle vs Stedoss actually took place.

I guess the comment could be moved to the Azerith vs Valentine match result since that was the match that didn't happen and allowed djsettle to skip RO64

Reply:
then

```suggestion
| **djsettle** ![][flag_GB] | **3** | 0 | ![][flag_GB] Stedoss | ¹ |
```

and add `1 - explanation` under the table, could be okay i think?

Reply:
Yeah seems like the sensible way to go, I'll do that.

Reply:
Ok, applied changes going to mark as resolved. For another time, there doesn't seem to be a set way to format `¹` explanations, so it might be worth considering for the style guide at some point.